mk/db.mk

```diff
@@ -49,20 +49,20 @@ db-migrate-seed: $(GO_OUTPUT)/dbmigrate ## Run dbmigrate seed
 
 .PHONY: db-down
 db-down: ## Stop postgres database
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container stop postgres
+	! $(DOCKER) container inspect $(DB_CONTAINER_NAME) &> /dev/null || $(DOCKER) container stop $(DB_CONTAINER_NAME)
 
 .PHONY: db-clean
 db-clean: db-down ## Clean database volume
-	! $(DOCKER) volume inspect postgres &> /dev/null || $(DOCKER) volume rm postgres
+	! $(DOCKER) volume inspect $(DB_CONTAINER_NAME) &> /dev/null || $(DOCKER) volume rm $(DB_CONTAINER_NAME)
 
 .PHONY: db-cli-connect
 db-cli-connect: ## Open a postgres cli in the container (it requires db-up)
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_PORT) password=$(DATABASE_PASSWORD)"
+	! $(DOCKER) container inspect $(DB_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DB_CONTAINER_NAME) psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_PORT) password=$(DATABASE_PASSWORD)"
```

Comment:
it looks like `db-cli-connect` and `db-dump-table` are using a default DATABASE_PORT variable, that is defined `varibles.mk`

Reply:
in this case that values is 5432, so it's causing an error

Reply:
good catch, i split that variable into internal and external port and used them appropriately.